# This is a message to hold data from the FeuerWeRR (Inertial Measurement Unit)

Header header
geometry_msgs/Vector3 imu_angular_velocity
geometry_msgs/Vector3 imu_linear_acceleration
geometry_msgs/Vector3 ma351_linear_acceleration
geometry_msgs/Vector3 magnetometer
time feuertime
